### PR TITLE
build(npm): Resolved all 'npm install' warnings.

### PR DESCRIPTION
**Pull request by mattrowsboats**
Updated turf package to only bring in what was used and added "license" and "repository" fields to
the package.json

### CODE REVIEW

File: tasking-mgr-projects/package.json [hunk 1]
```diff
@@ -6,13 +6,17 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/AmericanRedCross/osm-data-parse.git"
+  },
   "author": "danbjoseph",
-  "license": "",
+  "license": "ISC",
```
Comment (dalekunce) [inline tasking-mgr-projects/package.json]: Thanks for updating the package.json. However, all of the work that American Red Cross does is BSD-3-Clause.

Comment (mattrowsboats) [reply tasking-mgr-projects/package.json]: Ok!  Will make the change.  Thanks!

**Comment by danbjoseph** [comment]
thanks for this!

**Comment by mattrowsboats** [comment]
Absolutely!

**Pull request merged**